src/main/java/org/killbill/billing/client/KillBillHttpClient.java

```diff
@@ -325,7 +325,7 @@ public Response doOptions(final String uri, final Multimap<String, String> optio
     }
 
     private <T> T doPrepareRequestAndMaybeFollowLocation(final String verb, final String uri, final Object body, final Multimap<String, String> optionsRo, final Multimap<String, String> optionsForFollow, final int timeoutSec, final Class<T> clazz, final boolean followLocation) throws KillBillClientException {
-        final Multimap<String, String> options = HashMultimap.<String, String>create(optionsRo);
+        final Multimap<String, String> options = HashMultimap.create(optionsRo);
```

Comment:
I've tried compiling the code with Java 1.6.0_45 and it worked. Here's the Dockerfile I've used:
```docker
FROM jamesdbloom/docker-java6-maven

WORKDIR /usr/src/app

ADD . /usr/src/app
RUN mvn package
```